Lib/test/test_ast/test_ast.py

```diff
@@ -109,15 +111,15 @@ def cleanup():
 
         msg = "type object 'ast.AST' has no attribute '_fields'"
         # Both examples used to crash:
-        with self.assertRaisesRegex(AttributeError, msg):
+        with self.assertRaisesRegex(AttributeError, msg), self.assertWarns(DeprecationWarning):
```

Comment:
```suggestion
        with (
            self.assertRaisesRegex(AttributeError, msg),
            self.assertWarns(DeprecationWarning),
        ):
```

and ditto below, just to warp at 80 chars.